esi_leap/db/sqlalchemy/api.py

```diff
@@ -216,6 +215,31 @@ def offer_verify_availability(offer_ref, start, end):
                                                   end_time=end)
 
 
+def offer_verify_node_availability(resource_node_uuid, start, end):
```

Comment:
`offer_verify_resource_availability`; and maybe rename `offer_verify_availability` to `offer_verify_contract_availability`?

Reply:
fixed